cmake/modules/AddSwift.cmake

```diff
@@ -1384,6 +1389,15 @@ function(add_swift_library name)
           endif()
         endforeach()
 
+        set(swiftlib_framework_depends_flattened ${SWIFTLIB_FRAMEWORK_DEPENDS})
```

Comment:
Actually it didn't move.  It the diff not really knowing how I applied the changes.

What we have before was a loop to set swiftlib_module_dependency_targets, then later a loop to set swiftlib_framework_depends_flattened, and finally a loop for swiftlib_private_link_libraries_targets.

In the original code, swiftlib_private_link_libraries_targets was initialized to swiftlib_module_dependency_targets, but we don't want that any more. This is because swiftlib_module_dependency_targets now contains references to the swift module files, and we want swiftlib_private_link_libraries_targets to reference the linked images.  So I changed swiftlib_private_link_libraries_targets to initialise to empty instead of swiftlib_module_dependency_targets then added appends to this loop to set it appropriately.

And yes, I could definitely move the in a separate commit then only make the change the following a change in a new commit:

list(APPEND swiftlib_module_dependency_targets
              "swift${mod}${VARIANT_SUFFIX}")

to

list(APPEND swiftlib_module_dependency_targets
              "swift${mod}${MODULE_VARIANT_SUFFIX}")